### ISSUE TITLE
Can not run in batch size 1

### ISSUE THREAD

**Issue by leezonpen** [Jongchan/attention-module#5]
I try this module in detection task. When i try to make a inference, it throw an error. I found that it can not run forward in one batch size because of the nn.BatchNorm1d . How to deal with it?

**Comment by leezonpen** [comment]
I get it .
model.eval() should be used.
